src/components/PaymentLog.vue

```diff
@@ -902,6 +940,15 @@ export default {
         text += possible.charAt(Math.floor(Math.random() * possible.length))
       return text
     },
+    showRepaymentToggle() {
+      //if altarapay, display only slug has loan or when noBS (!altarapay product)
+      return this.isAltaraPay &&
```

Comment:
you don't need the tenary op, you can just return 

`return this.isAltaraPay && this.showRepaymentToggleList.includes( this.salesLogForm?.business_type_id?.slug)`